rollup/sync_service_test.go

```diff
@@ -18,6 +18,64 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+func setupLatestEthContextTest() (*SyncService, *EthContext) {
+	service, _, _, _ := newTestSyncService(false)
+	resp := &EthContext{
+		BlockNumber: uint64(10),
+		BlockHash:   common.Hash{},
+		Timestamp:   uint64(service.timestampRefreshThreshold.Seconds()) + 1,
+	}
+	setupMockClient(service, map[string]interface{}{
+		"GetLatestEthContext": resp,
+	})
+
+	return service, resp
+}
+
+// Test that if applying a transaction fails
+func TestSyncServiceContextUpdated(t *testing.T) {
+	service, resp := setupLatestEthContextTest()
+
```

Comment:
Could you add a check here for the initial values before calling `updateContext`? You can see that they are initialized from the database on start here: https://github.com/ethereum-optimism/go-ethereum/blob/9e1ca7e30a927df064ed9f0469a29f316c91a731/rollup/sync_service.go#L133

I'm guessing they are initialized to their zero values since `client.getEthContext` isn't being mocked out. On fresh boot, it gets the L1 blocknumber/timestamp of the deploy height of the CTC to guarantee determinism